modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -218,10 +219,8 @@ object GmosControllerEpics extends GmosEncoders {
   def setDisperser[T <: GmosController.SiteDependentTypes: Encoders](state: GmosCCEpicsState, d: T#Disperser): Option[IO[Unit]] = {
     val encodedVal = Encoders[T].disperser.encode(d)
     val s = applyParam(state.disperser.toUpperCase, encodedVal.toUpperCase, (_: String) => CC.setDisperser(encodedVal))
```

Comment:
Why don't we need `upperCase` when setting the value?

Reply:
I don't know, this just copy the code in the Tcl Seqexec. My guess is that the instrument changes the name in the status channel.